botfront/private/defaults/private.yaml

```diff
@@ -9,12 +9,12 @@ bfApiHost: http://botfront-api:8080
 endpoints:
```

Comment:
These were most certainly correct. Yours should be equivalent on MacOS, but incompatible with Linux environments.

Reply:
maybe I have something wrong somewhere else then? I don't get responses in the chat and conversations don't get saved with the old settings.

Reply:
fixed